extensions/wikia/Flags/tasks/FlagsExtractTemplatesTask.php

```diff
@@ -118,5 +118,13 @@ public function extractTemplatesFromPage( $pageId, Array $flagTypesToExtract ) {
 		}
 	}
 
+	public function migrateTemplatesToFlags( $template ) {
+		global $IP;
 
+		$wikiId = $this->getWikiId();
+
+		$cmd = "SERVER_ID={$wikiId} /usr/bin/php {$IP}/extensions/wikia/Flags/maintenance/MoveNotice.php --template='{$template}' --replaceTop --add --remove";
```

Comment:
Is `$template` potentially user controlled/defined? If so, this needs to be escaped with `wfEscapeShellArg` (same in MoveNotices.php).